docs/requirements-and-installation.md

```diff
@@ -65,16 +77,10 @@ Run these commands to verify:
 ```bash
 pip list
 check-devices-reachability.py --help
+which check-devices-reachability.py
 ```
 
-### Clone the repository and use `setup.py`
-
-Run these commands to clone the repository and to move to the new folder:
-
-```shell
-git clone https://github.com/arista-netdevops-community/network-test-automation.git
-cd network-test-automation
-```
+#### Using `python setup.py` commands
```

Comment:
Seems some commands are missing ?

Reply:
I messed it up with my git commands. will repush what is missing